### ISSUE TITLE
Font becomes weird when deployed

### ISSUE THREAD

**Issue by siddhugolu** [onweru/hugo-swift-theme#54]
Don't know whether this is something to do with my setup or what - but when I build the site locally, the fonts work as expected. But when I deploy it via Netlify, they are somewhat changed. Here's the screenshot comparison of the differences:

This is how it appears locally:

![image](https://user-images.githubusercontent.com/9106868/65437739-f66ba100-de41-11e9-9098-2cf529237190.png)

And this is what appears when deployed:

![image](https://user-images.githubusercontent.com/9106868/65437781-0be0cb00-de42-11e9-8edc-269a2d6b5f3a.png)

Some more screenshots of the content:
Local:

![image](https://user-images.githubusercontent.com/9106868/65438190-b0fba380-de42-11e9-909d-57775e364c2a.png)

Deployed:

![image](https://user-images.githubusercontent.com/9106868/65437853-2dda4d80-de42-11e9-8dd4-d109964a943e.png)

**Comment by onweru** [comment]
@siddhugolu, can you share the url where you deployed? I could have a look at it and find out what's what.

**Comment by siddhugolu** [comment]
@onweru, sure. It's thelazyoxymoron.me

**Comment by siddhugolu** [comment]
@onweru sure, it's [https://thelazyoxymoron.me/](url)

**Comment by onweru** [comment]
Ensure that `baseURL` in your file config.toml file is set to either __"https://thelazyoxymoron.me/"__ or __"/"__. It appears it's current set to __"localhost:1313"__.

See screenshot for example.

<img width="769" alt="Screen Shot 2019-09-24 at 7 27 53 AM" src="https://user-images.githubusercontent.com/16350351/65481308-6fe4ad00-de9d-11e9-906e-53a13256867c.png">

**Comment by siddhugolu** [comment]
It's set to that only, otherwise, I don't think the website can be live. Attaching the screenshot:

![image](https://user-images.githubusercontent.com/9106868/65482146-ad553480-deb5-11e9-9b97-596652def9e1.png)

**Comment by onweru** [comment]
Hmm,

Your site seems  to be requesting fonts assets from localhost:1313. Which is weird.

<img width="1255" alt="Screen Shot 2019-09-24 at 8 01 27 AM" src="https://user-images.githubusercontent.com/16350351/65482451-d5d33380-dea1-11e9-926e-3d0d948cd077.png">

<img width="1255" alt="Screen Shot 2019-09-24 at 8 06 06 AM" src="https://user-images.githubusercontent.com/16350351/65482580-382c3400-dea2-11e9-8cff-016ecb0bac6a.png">

1. How are you generating your site? Manually? through netlify?
2. May I have a look at your repo.

**Comment by siddhugolu** [comment]
Hmm, that's really weird. I'm generating my site through Netlify. Here's the repo: [https://github.com/siddhugolu/The-Lazy-Oxymoron](url)

**Comment by onweru** [comment]
Aha, I found the problem. Remove the `resources` dir. Also, add it to your `.gitignore` file. As you can see, [in this generated file](https://github.com/siddhugolu/The-Lazy-Oxymoron/blob/9cab1be98cd16920ba14d2cbb825d1f316fa5468/resources/_gen/assets/scss/scss/main.scss_65b0252c4a2fb050308e79c52b2c1909.content#L6-L41),  the generated resources contain `localhost:1313` as the base path.

As you do not commit generated resources, you will be fine 😉 😃

**Comment by siddhugolu** [comment]
Thanks for taking the time to dig through this! I searched around a bit and found a related issue with hugo: [https://github.com/gohugoio/hugo/issues/5226](url)

So basically, the problem was that I used to test my site using `hugo server -D` which when combined with `canonifyURLs=true` caused all the hyperlinks to be referenced locally. Solution? Did a simple `hugo` before committing the repo and it fixed the issue :)
